labs_all.json

```diff
@@ -105,5 +105,27 @@
     "yaml_dir": "labs/lab07-redactors/manifests",
     "k8s_installer_yaml_path": "labs/lab07-redactors/kurl-installer.yaml",
     "use_public_ip": true
+  },
+  {
+    "name": "Lab 8: Airgap Lite",
+    "slug": "lab08-airgap-lite",
+    "channel": "lab08-airgap-lite",
+    "channel_slug": "lab08-airgap-lite",
+    "customer": "lab08-airgap-lite",
+    "yaml_dir": "labs/lab08-airgap-lite/manifests",
+    "extra_releases": [
```

Comment:
Maybe also add a channel (used in case `promote` is true?)

Reply:
I like that idea if we ever end up needing it. Right now we don't so I won't add it. Maybe longer term we could just move both releases into a "releases" list